pandas/core/indexes/base.py

```diff
@@ -4504,7 +4511,7 @@ def delete(self, loc):
     def insert(self, loc, item):
         """
         Make new Index inserting new item at location. Follows
-        Python list.append semantics for negative values
+        Python list.append semantics for negative values.
```

Comment:
Do you mind adding the second sentence to another paragraph (with one blank line between the first).